pam/tools/pam-runner/pam-runner.go

```diff
@@ -58,6 +58,12 @@ func main() {
 	if coverDir := os.Getenv("GOCOVERDIR"); coverDir != "" {
 		defaultArgs = append(defaultArgs, "--exec-env", fmt.Sprintf("GOCOVERDIR=%s", coverDir))
 	}
+	if asanOptions := os.Getenv("ASAN_OPTIONS"); asanOptions != "" {
+		defaultArgs = append(defaultArgs, "--exec-env", fmt.Sprintf("ASAN_OPTIONS=%s", asanOptions))
+	}
+	if lsanOptions := os.Getenv("LSAN_OPTIONS"); lsanOptions != "" {
+		defaultArgs = append(defaultArgs, "--exec-env", fmt.Sprintf("LSAN_OPTIONS=%s", lsanOptions))
+	}
```

Comment:
We wanted to be as much as conservative as possible, so not pass anything to the pam program, unless that's explicitly defined.

It may not lead to problems, but also allows us to have a very controlled environment that works the same both in developer machines and in CI test environment.

Reply:
Ok, thanks for explaining! Not sure I agree with that but we don't have to discuss it here.

Reply:
You know, it's a library loaded by a PAM module that execs another binary, so the less we expose, the best.

Also, consider that `pam_exec.so` is exposing absolutely nothing if not the pam environment (which we've access through dbus anyways), so I feel it's better and as said allows us to be sure that everywhere we run this, it's behaving the same.